test/e2e/storage/csi_mock_volume.go

```diff
@@ -116,7 +116,7 @@ var _ = utils.SIGDescribe("CSI mock volume", func() {
 		m.provisioner = config.GetUniqueDriverName()
 
 		if tp.nodeSelectorKey != "" {
-			framework.AddOrUpdateLabelOnNode(m.cs, m.config.ClientNodeName, tp.nodeSelectorKey, f.Namespace.Name)
+			framework.AddOrUpdateLabelOnNode(m.cs, m.config.ClientNodeSelector[v1.LabelHostname], tp.nodeSelectorKey, f.Namespace.Name)
```

Comment:
@gnufied can you help review the mock driver test changes?

Reply:
sure

Reply:
@gnufied why does the attach limit test need to manually set labels on the node? I would like to remove this logic if possible.